onboard/catkin_ws/src/motion_planning/scripts/move_tasks.py

```diff
@@ -11,30 +11,34 @@ class MoveToPoseGlobalTask(Task):
     def __init__(self, x, y, z, roll, pitch, yaw, *args, **kwargs):
         super(MoveToPoseGlobalTask, self).__init__(*args, **kwargs)
 
-        self.desired_pose = Pose()
-        self.desired_pose.position.x = x
-        self.desired_pose.position.y = y
-        self.desired_pose.position.z = z
-        self.desired_pose.orientation = Quaternion(*quaternion_from_euler(roll, pitch, yaw))
+        self.desired_pose = Pose(
+                            position=Point(x=x, y=y, z=z),
+                            orientation=Quaternion(
+                                *quaternion_from_euler(roll, pitch, yaw)))
+        self.desired_twist = Twist()  # default 0 angular and linear velocity
 
     def _on_task_run(self):
         self.publish_desired_pose_global(self.desired_pose)
-        if task_utils.at_pose(self.desired_pose, self.state.pose.pose):
+        if task_utils.at_pose(self.state.pose.pose, self.desired_pose) and \
+                task_utils.at_vel(self.state.twist.twist, self.desired_twist):
             self.finish()
 
 
 class MoveToPoseLocalTask(MoveToPoseGlobalTask):
     """Move to pose given in local coordinates."""
 
     def __init__(self, x, y, z, roll, pitch, yaw, *args, **kwargs):
-        super(MoveToPoseLocalTask, self).__init__(x, y, z, roll, pitch, yaw, *args, **kwargs)
+        super(MoveToPoseLocalTask, self).__init__(
+            x, y, z, roll, pitch, yaw, *args, **kwargs)
```

Comment:
```suggestion
        super(MoveToPoseLocalTask, self).__init__(x, y, z, roll, pitch, yaw, *args, **kwargs)
```

Reply:
thanks for catching this. I originally assumed linter we are running had shorter max line and forgot to change this back